core/domain/exp_jobs_one_off.py

```diff
@@ -572,7 +572,7 @@ def map(exp_model):
         if not constants.DEV_MODE:
             exp_id = exp_model.id
             fs_old = fs_domain.AbstractFileSystem(
-                fs_domain.GcsFileSystem(exp_id))
+                fs_domain.GcsFileSystem(exp_id, None))
```

Comment:
Here and elsewhere, the args are wrong, right? The first arg should be an entity type and the second should be the entity ID?

EDIT: Ah, I see, this is for the old system. I suggest just deleting the three affected jobs. We won't need them any more after the migration, and leaving them in the codebase will just result in confusion.

Reply:
One of the affected jobs is my current job :), as it requires checking whether all images are present in both folders. I'll remove the other 2 jobs, but shall I remove my current job in another PR?

Reply:
I think you can remove the current job too, because we will be getting rid of the old folder anyway (right?). In fact I believe I have done that.

Reply:
Ok, will remove it then.

Reply:
Done